modules/cold-extraction/ColdDataRetriever.py

```diff
@@ -180,22 +196,6 @@ def retrieve():
                 subprocess.call("{0}/movescu -c {1} -b {2} -M PatientRoot -m PatientID={3} -m AccessionNumber={4} --dest {5}".format(DCM4CHE_BIN, SRC_AET, QUERY_AET, PatientID, Accession, DEST_AET), shell=True)
                 extracted_ones.append(temp_id)
 
-    # For the cases that have the EMPI.
-    elif (extraction_type == 'empi'):
```

Comment:
Can you please explain to me the reason why this code block is deleted?

Please make sure you are making changes based from the dev branch and not from the master branch.